src/mumble/ServerHandler.cpp

```diff
@@ -454,6 +462,8 @@ void ServerHandler::message(unsigned int msgType, const QByteArray &qbaMsg) {
 			ConnectionPtr connection(cConnection);
 			if (!connection) return;
 
+			iInFlightTCPPings -= 1;
```

Comment:
I don’t really like that we handle every ping message here in message, but not increase the counter in the counterpart `sendMessage`. But the current code does not seem to be symmetric at all, so that’s just how it is.